src/applications/personalization/profile-2/tests/components/personal-information/PersonalInformation.unit.spec.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+import { expect } from 'chai';
+import { setupServer } from 'msw/node';
+
+import { resetFetch } from 'platform/testing/unit/helpers';
+
+import * as mocks from '../../../msw-mocks';
+import {
+  getBasicContactInfoState,
+  renderWithProfileReducers,
+} from '../../unit-test-helpers';
+
+import PersonalInformation from '../../../components/personal-information/PersonalInformation';
+
+function createBasicInitialState() {
+  return {
+    scheduledDowntime: {
+      globalDowntime: null,
+      isReady: true,
+      isPending: false,
+      serviceMap: {
+        get() {},
+      },
+      dismissedDowntimeWarnings: [],
+    },
+    vaProfile: {
+      personalInformation: {
+        gender: 'M',
+        birthDate: '1986-05-06',
+      },
+    },
+    user: {
+      profile: {
+        vet360: getBasicContactInfoState(),
+      },
+    },
+  };
+}
+
+function fillOutAndSubmitBankInfoForm(view) {
```

Comment:
Is this logic related to Direct Deposit supposed to live in a separate test file and actually being called there?

Reply:
This was my being sloppy :) Thanks